### ISSUE TITLE
Release trained models on Hugging Face

### ISSUE THREAD

**Issue by NielsRogge** [rohitgandikota/erasing-llm#3]
Hello @rohitgandikota 🤗 

I'm Niels and work as part of the open-source team at Hugging Face. I discovered your work through AK's daily papers: https://huggingface.co/papers/2410.02760. The paper page lets people discuss about your paper and lets them find artifacts about it (your model for instance) you can also claim the paper as yours which will show up on your public profile at HF.

Would you like to host the model you've pre-trained on https://huggingface.co/models? Hosting on Hugging Face will give you more visibility. We can add tags in the model cards so that people find the models easier, link it to the paper page, etc.

If you're down, leaving a guide [here](https://huggingface.co/docs/hub/models-uploading). After uploaded, we can also link the models to the paper page (read [here](https://huggingface.co/docs/hub/en/model-cards#linking-a-paper)) so people can discover your model.

Kind regards,

Niels

**Comment by rohitgandikota** [comment]
Done! Uploaded here: https://huggingface.co/collections/baulab/elm-6715d68576da0cd1a89c0c04

**Comment by NielsRogge** [comment]
Awesome, thanks a lot 🤗

**Comment by anaekin** [comment]
@rohitgandikota Would it be possible to upload the model Llama-2-7B for harry potter concept erasure? I tried to train myself, but I didn't get the same HP-MCQ accuracy as the paper.
